sdk/go/common/resource/plugin/plugin.go

```diff
@@ -159,7 +159,7 @@ var errPluginNotFound = errors.New("plugin not found")
 func dialPlugin[T any](
 	portNum int,
 	bin string,
-	prefix string,
+	name string,
```

Comment:
In this file, we rename `prefix` to `name` for clarity, as it seems to serve as the plugin's (caller-supplied) name.  One could use "label" or "displayName" but I felt that "name" is sufficiently clear.